app/src/main/res/layout/edit_prof_activity.xml

```diff
@@ -67,10 +64,18 @@
         android:id="@+id/linearLayout4"
         android:layout_width="match_parent"
         android:layout_height="101dp"
-        android:layout_marginBottom="240dp"
+        android:layout_marginStart="8dp"
```

Comment:
if all the margins are 8 you can change this to

android:layout_margin="8dp" and remove all the other lines